baby-liminal-extension/src/lib.rs

```diff
@@ -0,0 +1,111 @@
+#![cfg_attr(not(feature = "std"), no_std)]
+
+#[cfg(feature = "ink")]
+pub mod ink;
+
+#[cfg(feature = "substrate")]
+pub mod substrate;
+
+#[cfg(feature = "substrate")]
+pub mod executor;
+
+#[cfg(feature = "ink")]
+use ::ink::prelude::vec::Vec;
+#[cfg(feature = "substrate")]
+use obce::substrate::sp_std::vec::Vec;
+use scale::{Decode, Encode};
+#[cfg(feature = "std")]
+use scale_info::TypeInfo;
+
+// `pallet_baby_liminal::store_key` errors
+pub const BABY_LIMINAL_STORE_KEY_TOO_LONG_KEY: u32 = 10_001;
+pub const BABY_LIMINAL_STORE_KEY_IDENTIFIER_IN_USE: u32 = 10_002;
+pub const BABY_LIMINAL_STORE_KEY_ERROR_UNKNOWN: u32 = 10_003;
```

Comment:
nit: What I like to do personally is to use a "base" for the error group and then offset it for specific errors:
```rust
const BABY_LIMINAL_STORE_KEY_ERROR: u32 = 10_000;
pub const BABY_LIMINAL_STORE_KEY_TOO_LONG_KEY: u32 = BABY_LIMINAL_STORE_KEY_ERROR + 1;
// etc.
const BABY_LIMINAL_VERIFY_ERROR: u32 = 11_000;
pub const BABY_LIMINAL_VERIFY_DESERIALIZING_PROOF_FAIL: u32 = BABY_LIMINAL_VERIFY_ERROR + 1;
// etc.
```
this allows for easier range checks (if necessary):
```rust
fn is_baby_liminal_store_key_error(error_code: u32) -> bool {
  error_code > BABY_LIMINAL_STORE_KEY_ERROR && error_code < BABY_LIMINAL_VERIFY_ERROR
}
```

Reply:
Thanks, fixed!